src/formats/simplenote.py

```diff
@@ -21,8 +21,8 @@ def convert(self, file_or_folder: Path):
         )
 
         for note_simplenote in input_json["activeNotes"]:
-            # title is the first line
-            title, body = note_simplenote["content"].split("\r\n", maxsplit=1)
+            # title is the first line. In case of title-only notes, create empty second line
+            title, body = title, body = (note_simplenote["content"].split("\r\n", maxsplit=1)+[""])[:2]
```

Comment:
```suggestion
            title, body = common.split_h1_title_from_body(note_simplenote["content"])
```

Thanks for the PR! Actually, there is already a function that splits the title from the body in a more robust way. I guess it's better to use it to avoid duplicated code.

Reply:
Thanks for suggestion, I missed that function. With the fix to handle title-only notes, the tool flawlessly imported my about 300 notes from simplenote, making the whole migration quick and painless

Reply:
I'm glad to hear that it was useful to you. Thanks for the feedback :)